static/bidder-info/zmaticoo.yaml

```diff
@@ -0,0 +1,9 @@
+endpoint: "https://bid.zmaticoo.com/prebid/bid"
```

Comment:
`https` endpoint is not reachable 
```
curl --location --request POST 'https://bid.zmaticoo.com/prebid/bid'
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

Reply:
It may be a problem with your local root certificate. I tested it on my server and it was ok.
<img width="1196" alt="image" src="https://github.com/prebid/prebid-server/assets/8509023/52bd32af-efa4-4a5c-b1c8-4bd9a60a9e3a">

Reply:
@lxj15398019970  are their any geographical restriction to curl endpoint?

Reply:
no，where is your region ? what is the op system？This is a global domain。

Reply:
@lxj15398019970  we have checked within our team. We are still facing `SSL certificate problem` on multiple machines.

Reply:
i am sorry，We've found the problem and fixed it，please try again。

Reply:
```
 curl -i --location --request POST 'https://bid.zmaticoo.com/prebid/bid'
HTTP/2 204
date: Tue, 09 Jan 2024 12:03:40 GMT
error: 1_0
access-control-allow-origin: *
access-control-allow-headers: Content-Type
strict-transport-security: max-age=15724800; includeSubDomains
```
 endpoint is reachable